Stream_support/include/CGAL/IO/binary_file_io.h

```diff
@@ -48,7 +49,7 @@ I_Binary_write_float32(std::ostream& out, float f) {
 }
 inline void
 I_Binary_write_bool(std::ostream& out, bool b) {
-    char c = static_cast<char>(b);
+    char c = (b ? 1 : 0);
```

Comment:
If you really want to avoid conversion warnings, then you should use the character literals:
```suggestion
    char c = (b ? '\1' : '\0');
```

I am actually surprised that `char c = static_cast<char>(b);` can give any warning at all.

Reply:
It does not, but it seemed more consistent with the reading part (which does raise a warning if you do `bool b = static_cast<bool>(c)`).

Reply:
That `bool b = static_cast<bool>(c);` emits a warning is also very surprising. What was the warning?

Reply:
`C:\CGAL\test\CGAL-5.2-Ic-20\include\CGAL/IO/binary_file_io.h(78): warning C4800: 'char': forcing value to bool 'true' or 'false' (performance warning)`

Reply:
Shall we add it  in   `<CGAL/disable_warnings.h>`

Reply:
No: you see that the fix is trivial. `<CGAL/disable_warnings.h>` was a mistake and we should try to reduce it, instead of increasing its size. `<CGAL/disable_warnings.h>` was supposed to be used to remove warnings emitted from third-party libraries.

And, anyway, C4800 is already in it:
https://github.com/CGAL/cgal/blob/b8d5c5e89a28f73a2e59086ad36853c03e1c1347/Installation/include/CGAL/disable_warnings.h#L13-L28

There are very useful warnings in that list. For example:
 - C4706: assignment within conditional expression